docs/technical_reference/flowsheets/electrodialysis_1stack_conc_recirc.rst

```diff
@@ -0,0 +1,116 @@
+One-stack electrodialysis with a concentrate fluid recirculation
+================================================================
+
+Introduction
+------------
+Electrodialysis (ED) is a promising technology for desalinating brackish waters and has been deployed at industrial scales [1]_. It utilizes electrical potential to drive ion diffusion through anion and cation exchange membranes, resulting in the dilution of the feed stream while producing a concentrated brine. A single ED stack is an assembly of multiple flow-by cells separated by alternating cation and anion exchange membranes positioned between a pair of electrodes. When a voltage is applied, ions in the cell pair are driven from one channel (forming the diluate channel) to the other (forming the concentrate channel), thereby desalinating water. A one-stack ED desalination system represents a basic ED operation from which more complicated or larger-scale systems can be derived. Analyzing a one-stack ED system therefore provides valuable information on the technology's performance and cost-effectiveness for a given treatment task.
+
+Equal flow conditions through the diluate and concentrate channels would result in a product water recovery of 50%. Larger water recoveries are commonly achieved by operating the system in a feed-and-bleed mode, where a portion of the concentrate outlet is recirculated back to its inlet, thus increasing the portion of product water from the diluate outlet. This flowsheet simulates a one-stack ED system operated in feed-and-bleed mode.  A simpler ED flowsheet without fluid recirculation is presented as 
+
+     * `One-Stack Electrodialysis <https://watertap--1386.org.readthedocs.build/en/1386/technical_reference/flowsheets/electrodialysis_1stack.html>`_.
```

Comment:
This link references the build from the PR instead of latest.
```suggestion
     * `One-Stack Electrodialysis <https://watertap.readthedocs.io/en/latest/technical_reference/flowsheets/electrodialysis_1stack.html>`_.
```

You can also check the discussion here between Marcus and I for other ways to link to other documentation. https://github.com/watertap-org/watertap/pull/1396/files/5ec96d06e3f6df729e71b9cb1e7c3a3a32da6015#diff-1009f2493fb3cb501cbd2ac5af69180ca4ba15b279a71519ed0fbe636de9dd14R14